### DISCUSSION TITLE
PyApp packaging of your own project

### DISCUSSION THREAD

**Discussion by trappitsch** [ofek/pyapp#104] [Show and tell]
I really like PyApp and enjoy the feature it offers. In order to make packaging some of my own tools easier, I stitched together a CLI named `box`. Maybe somebody is interested in it, here are [docs](https://box.rtfd.io) and here the [repo](https://github.com/trappitsch/box). This is very much work in progress but the following now works:

- Initialize your project (requires a `pyproject.toml`) with box
- Package automatically with PyApp
- Rename and move the executable into `target/release/your-project`

I thought this might be useful for some people. More features to come, see [issues on the repo page](https://github.com/trappitsch/box/issues). Of course, if you have any ideas, please chime in!

An example on how to use it can be found [here](https://box.readthedocs.io/en/latest/examples/). 

Thanks again @ofek for the amazing capabilities of PyApp!

**Comment by ofek** [comment]
That's awesome!
